datasets/loopfietsnetwerk/dataset.json

```diff
@@ -125,7 +125,7 @@
               "format": "date",
               "description": "datum creatie dataset"
             },
-            "geometry": {
+            "geometrie": {
```

Comment:
Als de geometrie velden een andere naam hebben dan "geometry" is een mainGeometry property nodig. https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#geometry

Reply:
Je kan `  "mainGeometry": "geometrie",` toevoegen op tabel niveau

Reply:
Wel vreemd dat de validator dit niet automatisch heeft opgepikt.